src/components/Form/UploadDocumentForm.jsx

```diff
@@ -84,47 +96,99 @@ const UploadDocumentForm = () => {
       statusType="Upload status"
       defaultMessage="To be uploaded..."
     >
-      <form onSubmit={handleDocumentUpload} autoComplete="off">
-        <label htmlFor="verify-checkbox">
-          Verify File on upload:
-          <input
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleDocumentUpload} autoComplete="off">
+          <FormControlLabel
+            control={<Checkbox />}
+            label="Verify file on upload"
             id="verify-checkbox"
-            type="checkbox"
             value={state.verifyFile}
-            onClick={() => dispatch({ type: 'TOGGLE_VERIFY_FILE' })}
+            onChange={() => dispatch({ type: 'TOGGLE_VERIFY_FILE' })}
           />
-        </label>
-        <div style={formRowStyle}>
           <DocumentSelection
             htmlForAndIdProp="upload-doc"
             handleDocType={handleDocType}
             docType={docType}
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
           <br />
+          <FormControl>
+            {/* TODO: Determine whether InputLabel below is necessary or redundant */}
+            <InputLabel htmlFor="upload-doc-expiration" />
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                id="upload-doc-expiration"
+                name="date"
+                format="MM/DD/YYYY"
+                label="Expiration Date"
+                value={expireDate}
+                onChange={(newExpireDate) => setExpireDate(newExpireDate)}
+                type="date"
+                slotProps={{
+                  textField: {
+                    helperText: 'MM/DD/YYYY'
+                  }
+                }}
```

Comment:
Same as in CrossPodWriteForm, think you could replace Lines 126 and 130 with just `<FormHelperText>MM/DD/YYYY</FormHelperText>` after  `<DatePicker>`:

```
<FormControl>
  <LocalizationProvider>
    <DatePicker />
    <FormHelperText>MM/DD/YYYY</FormHelperText>
  </LocalizationProvider>
</FormControl>
```